### ISSUE TITLE
Type issue with applying multiple scopes 

### ISSUE THREAD

**Issue by antonyarchukcodemotion** [RobinBuschmann/sequelize-typescript#353]
I have a few scopes on my model
```
@Scopes({
  accountAndSource: (source: string, accountId: string) => ({
    where: {
      source,
      accountId,
    },
  }),
  listAttributes: {
    attributes: ['id', 'originalId', 'name'],
  },
})
```

Now I'd like to apply both at the same time. 
I'm trying to do 
```
Client.scope('listAttributes', {
          method: ['accountAndSource', 'someSystemName', 'someID'],
        }).findAll(...
```
It's ok by sequelize docs and it actualy works, but I'm getting an error by compiler: 
`Error:(27, 40) TS2345: Argument of type '{ method: string[]; }' is not assignable to parameter of type 'string'.`

Looks like the declaration for `.scope` is a list of scopes w/o attributes (string[]) or only one scope (object).  

Any ideas how I can use it without errors? 
Thanks in advance!

**Comment by RobinBuschmann** [comment]
@antonyarchukcodemotion thanks for reporting. This was fixed with #322. But need to be published.

**Comment by RobinBuschmann** [comment]
Is published now as beta release: `sequelize-typescript@0.6.4-beta.0`
